Official builds failing during signing validation

Failing at 100% since at least 2020/2/21 with a single signing validation issue. 

https://dev.azure.com/dnceng/internal/_build?definitionId=679&_a=summary

```
D:\a\1\s\.packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.20117.3\tools\SdkTasks\SigningValidation.proj(57,5): error : Signing validation failed. Check D:\a\1\s\artifacts\log\Debug\\signcheck.errors.log for more information.
##[error].packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.20117.3\tools\SdkTasks\SigningValidation.proj(57,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Signing validation failed. Check D:\a\1\s\artifacts\log\Debug\\signcheck.errors.log for more information.
```

The signing log only had the following interesting info that I could find

```
    [File] WixDepCA, Signed: False, Excluded [Error] HRESULT: 800b0100 (No signature was present in the subject)
    [File] WixUIWixca, Signed: False, Excluded [Error] HRESULT: 800b0100 (No signature was present in the subject)
```

--- a/eng/Configurations.props
+++ b/eng/Configurations.props
@@ -1,7 +1,7 @@
 <Project>
   <!-- Honor the generic RuntimeConfiguration property. -->
   <PropertyGroup>
-    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(Configuration)</RuntimeConfiguration>
+    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(ConfigurationGroup)</RuntimeConfiguration>
     <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == '' AND ('$(Configuration)' == 'Debug' OR '$(Configuration)' == 'Release')">$(Configuration)</RuntimeConfiguration>
     <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">Debug</RuntimeConfiguration>
     <CoreCLRConfiguration Condition="'$(CoreCLRConfiguration)' == ''">$(RuntimeConfiguration)</CoreCLRConfiguration>

--- a/eng/build.ps1
+++ b/eng/build.ps1
@@ -138,7 +138,7 @@ foreach ($argument in $PSBoundParameters.Keys)
     "build"                { $arguments += " -build" }
     "buildtests"           { if ($build -eq $true) { $arguments += " /p:BuildTests=true" } else { $arguments += " -build /p:BuildTests=only" } }
     "test"                 { $arguments += " -test" }
-    "configuration"        { $arguments += " -configuration $((Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])))" }
+    "configuration"        { $configuration = (Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])); $arguments += " /p:ConfigurationGroup=$configuration -configuration $configuration" }
     "runtimeConfiguration" { $arguments += " /p:RuntimeConfiguration=$((Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])))" }
     "framework"            { $arguments += " /p:BuildTargetFramework=$($PSBoundParameters[$argument].ToLowerInvariant())" }
     "os"                   { $arguments += " /p:OSGroup=$($PSBoundParameters[$argument])" }

--- a/eng/build.sh
+++ b/eng/build.sh
@@ -94,7 +94,7 @@ while [[ $# > 0 ]]; do
       ;;
      -configuration|-c)
       val="$(tr '[:lower:]' '[:upper:]' <<< ${2:0:1})${2:1}"
-      arguments="$arguments -configuration $val"
+      arguments="$arguments /p:ConfigurationGroup=$val -configuration $val"
       shift 2
       ;;
      -framework|-f)

